src/model/entities/action.ts

```diff
@@ -1,6 +1,12 @@
 import { ActionType, PartActionType, PieceActionType } from '../enums/action-type'
 import { PartInterface } from './part'
 import { Piece, PieceInterface } from './piece'
+import { PieceType } from '../enums/piece-type'
+
+export interface ActionManifest {
+  actionType: PieceType
```

Comment:
`actionType` is not the same as `PieceType`. ActionTypes are `PartAction` and `PieceAction`.

Reply:
Renamed actionType to pieceType.